test/fixtures/html/integration/_types/things/index.html

```diff
@@ -0,0 +1,126 @@
+                        <path d="M95.5 56.3C95.5 57.9 96.2 58.3 97.1 58.3 97.6 58.3 98 58.2 98.4 58L98.4 61.8C97.5 62.2 96.6 62.4 95.6 62.4 93 62.4 91.4 61.3 90.8 58.8 89.7 61.6 87.2 62.6 84.3 62.6 80.4 62.6 77.7 60.3 77.7 56.5 77.7 51.6 81.6 48.8 90.5 49.3L90.5 46.8C90.5 44.3 89.1 43.2 87 43.2 86 43.1 85.1 43.5 84.4 44.1 83.7 44.8 83.3 45.7 83.4 46.7L78.5 45.9C78.8 41.1 82.9 39.3 87.3 39.3 92.1 39.3 95.5 41.3 95.5 46.3L95.5 56.3ZM90.5 52.3C85 52.2 82.8 53.5 82.8 56 82.7 56.8 83.1 57.6 83.7 58.2 84.3 58.7 85.1 59 85.9 58.9 88 58.9 90.5 57.7 90.5 55.2L90.5 52.3Z"></path>
+                        <path d="M125.6 39.3C129.3 39.3 132 41.3 132 46.1L132 62 127 62 127 47.8C127 45.2 126 43.6 123.6 43.6 121.2 43.6 119 45.2 119 49.4L119 62 114 62 114 47.8C114 45.2 113 43.6 110.5 43.6 108.1 43.6 106 45.2 106 49.5L106 62 101 62 101 39.8 105 39.8 105.6 44.1C106.5 41.1 109.4 39.1 112.5 39.3 115.6 39.3 117.9 40.6 118.5 44.2 119.7 40.7 122.4 39.3 125.6 39.3Z"></path>
+                        <path d="M150 55.2L154.4 56.5C153.4 61.1 149.6 62.6 145.3 62.6 139.2 62.6 135.1 58.5 135.1 51.2 135.1 43.7 139.1 39.3 145.5 39.3 151.4 39.3 154.5 43.3 154.5 48.9 154.5 49.9 154.4 50.9 154.1 51.9L140 51.9C140.2 55.9 142.1 58.7 145.5 58.7 147.7 58.9 149.6 57.4 150 55.2ZM149.6 48.6C149.6 48.3 149.6 48 149.6 47.7 149.7 46.5 149.2 45.3 148.4 44.5 147.6 43.6 146.4 43.1 145.2 43.1 142.3 43.1 140.4 45.2 140.1 48.6L149.6 48.6Z"></path>
+                        <path d="M162.4 44.1C163.6 40.8 166.2 39.3 169.5 39.3 173.1 39.3 176.2 41.3 176.2 46.2L176.2 62 171.2 62 171.2 47.9C171.2 45.2 170.2 43.6 167.6 43.6 165.1 43.6 162.8 44.9 162.8 49.5L162.8 62 157.8 62 157.8 39.8 161.9 39.8 162.4 44.1Z"></path>
+                        <path d="M194.3 61.2C192.5 62.2 190.4 62.6 188.4 62.6 184.3 62.6 181.7 60.5 181.7 56L181.7 43.8 178.5 43.8 178.5 39.8 181.7 39.8 182.9 33.7 186.8 33.7 186.8 39.8 193.4 39.8 193.4 43.8 186.8 43.8 186.8 55C186.8 57.4 187.8 58.3 189.6 58.3 191 58.3 192.3 57.8 193.4 57L194.3 61.2Z"></path>
+                        <path d="M49.1 45.9L48.5 39.8 44.2 39.8 44.2 62 49.2 62 49.2 51.1C49.2 46.7 52.2 44.7 55.2 44.7L55.9 44.7 55.9 39.4C51.9 39.8 49.9 42.4 49.1 45.9Z"></path>
+                        <path d="M65.5 58C65 58.1 64.5 57.9 64.1 57.5 63.7 57.2 63.5 56.6 63.5 56.1L63.5 31.2 58.5 31.2 58.5 57.3C58.5 60.9 61 62.4 64 62.4 65 62.4 66 62.2 67 61.9L67 57.9C66.5 58 66 58 65.5 58Z"></path>
+                    </g>
+                </svg></a></div>
+    </header>
+    <main>
+        <div class="section--primary" id="content">
+            <div class="container">
+                <h1><span>Things</span></h1>
+            </div>
+        </div>
+        <section class="section--major section--wide ">
+            <div class="container">
+                <div class="status--highlight ">
+                    <p>There is currently no prototypes to show on this page.</p>
```

Comment:
Change to something more generic to use everywhere. 'There are currently no views'